workflow/snakemake/Snakefile

```diff
@@ -0,0 +1,85 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ cd reana-demo-cms-h4l
+#   $ mkdir snakemake-local-run
+#   $ cd snakemake-local-run
+#   $ virtualenv ~/.virtualenvs/cms-h4l-snakemake
+#   $ source ~/.virtualenvs/cms-h4l-snakemake/bin/activate
+#   $ pip install snakemake
+#   $ cp -a ../code ../data .
+#   $ snakemake -s ../workflow/snakemake/Snakefile -p --cores 4 --use-singularity
+
+
+
+rule all:
+    input:
+        "results/mass4l_combine_userlvl3.pdf"
+
+
+rule analyze_data:
+    input:
+        "data",
+        "code"
+    output:
+        "results/DoubleMuParked2012C_10000_Higgs.root"
+    container:
+        "docker://cmsopendata/cmssw_5_3_32"
+    shell:
+        "source /opt/cms/cmsset_default.sh "
+        "&&  scramv1 project CMSSW CMSSW_5_3_32 "
```

Comment:
Please remove extra blank spaces in front of `scramv1`, here and elsewhere in the Snakefile.